### DISCUSSION TITLE
Welcome to ZOAU Discussions!

### DISCUSSION THREAD

**Discussion by ijmitch** [IBM/zoau-samples#16] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

⚠️ Note that this is an **unofficial** support forum.  If you have a ZOAU license and need assistance, please contact your IBM representative and open up a case.

**Comment by ijmitch** [comment, marked as answer]
# Ian Mitchell

Colleague of @MikeFultonDev at IBM and the one to suggest using the Discussions feature of Github since that's what the README for this project says we do here!

Mike granted me admin permissions to turn Discussions on, so I got to make the first 'Welcome' post as well.

**Comment by EnsonoStevenPerva** [comment]
# Steven Perva
I have been a z/OS Systems Programmer since I graduated college in 2010. I have experience across most walks of IT including distributed and cloud infrastructure as well as software development languages from HLASM to C#.

I have been using ZOAU as a component for process automation with my current employer, Ensono, since I started here in 2021. I hope to be engaged with the community in sharing ideas, solutions, and challenges to overcome as Z becomes more platform agnostic and hybrid.

**Comment by MikeFultonDev** [comment]
Here's a short blurb [about me](https://about.me/mike_fulton)
I'm a big fan of ZOAU - I wrote some of the initial code, in particular mvscmd which kind of kick-started all of this for me.

**Comment by ddimatos** [comment]
Little about me; been at IBM for 15 years, started my career in open source kernel development, then ended up at IBM in Systems doing middle-ware development, then full stack development, then shifted to distributed computing to lead some components of Hadoop (HDFS, Mapreduce, Yarn), later Spark and now Ansible Core for Z and also in my squad is ZOAU, hence my interest here. :)
